ui.apps/src/main/content/jcr_root/apps/acs-commons/touchui-widgets/composite-multifield/.content.xml

```diff
@@ -4,7 +4,7 @@
           jsProcessor="[default:none,min:gcc]"
           allowProxy="{Boolean}true"
           categories="[acs-commons.authoring.composite-multifield,acs-commons.granite.ui.coral2.foundation]"
-          dependencies="[underscore]"/>
+          dependencies="[acs-commons.lodash]"/>
```

Comment:
We need to update lodash, or are you sure that we are not affected by any of those vulnerabilities: https://snyk.io/vuln/npm:lodash?

Reply:
Just for what it's worth, the version of lodash here is the one that used to be in AEM (e.g. in 6.4) so it isn't going backwards in security in those lower environments. Perhaps lodash was removed from 6.5 for security reasons? I haven't checked into why.

Reply:
And, in fact, there is still this version of lodash in AEM at `/libs/clientlibs/granite/lodash/underscore` with the category `lodash.underscore`

Reply:
Then we should rather rely on the clientlib shipped with AEM as Adobe takes care of fixing vulnerabilities then. To still be compatible with AEM 66.4 we probably need to depend on both lodash categories.

Reply:
In the commit it is lodash.underscore for all 3 components so i expect that to suffice.
Verified the existence of /libs/clientlibs/granite/lodash/undescore in 6.3 as well so we are backward compatible in the support window (6.2 is no longer under support)